sdk/monitor/monitor-opentelemetry-exporter/src/utils/logUtils.ts

```diff
@@ -120,7 +120,7 @@ function createTagsFromLog(log: ReadableLogRecord): Tags {
 }
 
 function createPropertiesFromLog(log: ReadableLogRecord): [Properties, Measurements] {
```

Comment:
It would be good to separate this a different method as well, right now is only applicable for Legacy Application Insights so no need to be calling this for all other logs, we can also add support it for Otel logs in the future easier.

Reply:
Pulled this out as its own method called only on legacy application insights logs. In any other cases, measurements will be populated with an empty object.